### ISSUE TITLE
Convert anything with an `onClick` handler to be a button

### ISSUE THREAD

**Issue by christiangenco** [Taoula/firebaseui-react#3]
For accessibility, any code with an `onClick` prop like this:

```jsx
<p onClick={() => setResetPassword(false)}>
  {`<- `}Go back
</p>
```

should be a button. This makes the mouse a pointer by default and makes it easier for screen readers to use the component.
